5calls/app/src/main/java/org/a5calls/android/a5calls/net/OkHttp3Stack.java

```diff
@@ -0,0 +1,158 @@
+package org.a5calls.android.a5calls.net;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.Request;
+import com.android.volley.toolbox.HttpStack;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import okhttp3.Call;
+import okhttp3.Headers;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Protocol;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.ProtocolVersion;
+import org.apache.http.StatusLine;
+import org.apache.http.entity.BasicHttpEntity;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.message.BasicHttpResponse;
+import org.apache.http.message.BasicStatusLine;
+
+/**
+ * Taken from: https://gist.github.com/NightlyNexus/bdacc5c6951fd9a82b01695e6f6bc600
```

Comment:
Cool. Is there an open-source license on that gist that we can add to our licenses information?

Reply:
Done. See:

https://github.com/5calls/android/pull/89/files?diff=split#diff-69d7ccf990174d7623a55dece091005cR58
https://github.com/5calls/android/pull/89/files?diff=split#diff-d2c7e22841894f0bd2a901ab4e0dce39R4